crates/wasm/src/lib.rs

```diff
@@ -15,21 +17,72 @@ pub struct BuildInfo {
     dirty: &'static str,
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct WASMCompileOptions {
+    // Compile each contract function used within the program
+    #[serde(default = "bool::default")]
```

Comment:
Is the `= "bool::default"` needed here?

Reply:
yes, to default if not provided as an option